blocks/header/header.css

```diff
@@ -10,8 +10,8 @@ header nav {
   box-sizing: border-box;
```

Comment:
I will likely rework most of these styles next sprint

Reply:
Do you mean for the work in [ADB-157](https://sparkbox.atlassian.net/browse/ADB-157) and [ADB-160](https://sparkbox.atlassian.net/browse/ADB-160)? I can hold off on looking at this file too closely, then!